### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by forLoop94**
1. I used media queries in my CSS file to make my desktop version look like the Figma template I selected (Desktop: >=768px).

2. I used Grid and Flexbox to lay out the elements on the page.

3. I tried to replicate the design as much as possible. 

Thank you.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -56,6 +57,10 @@ header {
   background-color: #acb7c3;
 }
 
+.desktop-nav {
+  display: none;
+}
```
Comment (OyePriscilla) [inline styles.css:62]: - [ ] I observed that you repeated this line of code a lot, this is not the best according to CSS best practices. Kindly implement it the way you did in line 189 -190 for all the repeated code. 👍
**Note: Please kindly look through lines (147, 172, 190, 406, 472, 566) of code carefully. 👍**

File: styles.css [hunk 2]
```diff
@@ -525,3 +551,308 @@ form {
...
+
+.form-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .right-side-image {
+    display: none;
+  }
```
Comment (OyePriscilla) [inline styles.css:573]: - [ ] I observed that you repeated this line of code a lot, this is not the best according to CSS best practices. Kindly implement it the way you did in line 189 -190 for all the repeated code. 👍
Note: Please kindly look through lines (596, 691, 708, 743, 777, 797,815 and 851) of code carefully. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @forLoop94 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Use CSS media query ✔️
Use Grid and Flexbox to layout element on the page ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by forLoop94** [comment]
1. The hand cursor has been implemented on all cards. I have added a hover state to all cards.
2. All sections not responding well to media queries have been fixed. 
3. All buttons now have button interactions for all states listed in the Figma design.
4. The "grid-container-desktop" section has been deleted. Contents of "grid-container" were used for both versions. Your material was very helpful. Thank you.
5. Inline styling has been removed for "works-card-desktop" image and taken to the external CSS file.
6. Inline styling has been removed for "desktop-divider" image and taken to the external CSS file.
7. All repeated lines of code have been grouped into single statements.

Thank you.

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @forLoop94 ,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by forLoop94** [comment]
Thank you @Favourezeugwa

**Comment by OyePriscilla** [comment]
> 1. The hand cursor has been implemented on all cards. I have added a hover state to all cards.
> 2. All sections not responding well to media queries have been fixed.
> 3. All buttons now have button interactions for all states listed in the Figma design.
> 4. The "grid-container-desktop" section has been deleted. Contents of "grid-container" were used for both versions. Your material was very helpful. Thank you.
> 5. Inline styling has been removed for "works-card-desktop" image and taken to the external CSS file.
> 6. Inline styling has been removed for "desktop-divider" image and taken to the external CSS file.
> 7. All repeated lines of code have been grouped into single statements.
> 
> Thank you.

@forLoop94 I'm so glad you got it well. 👍

**Comment by forLoop94** [comment]
@OyePriscilla I'm glad too, but all thanks to you. Your suggestions were clear and you provided very helpful materials. Thank you.

**Pull request merged**